projects/testing-library/src/lib/testing-library.ts

```diff
@@ -116,13 +116,12 @@ export async function render<SutType, WrapperType = SutType>(
     fixture.componentRef.injector.get(ChangeDetectorRef).detectChanges();
   };
 
-  let router = routes ? inject(Router) : null;
   const zone = inject(NgZone);
-  const navigate = async (elementOrPath: Element | string, basePath = ''): Promise<boolean> => {
-    if (!router) {
-      router = inject(Router);
-    }
 
+  const router = inject(Router);
+  router?.initialNavigation();
```

Comment:
I see, thanks!

Reply:
It will be available when the PR is merged (now) 😀